lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -0,0 +1,12 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi < DelegateClass(ActiveModelSerializers::Adapter::JsonApi)
+        def initialize(serializer, options = {})
+          warn "Calling deprecated ActiveModel::Serializer::Adater::JsonApi in #{caller[0..2].join(', ')}. Please use ActiveModelSerializers::Adapter::JsonApi"
```

Comment:
- typo: `s/Adater/Adapter`
- Does this work? `Calling deprecated #{self.class.name} (#{__FILE__}) from #{caller[0..2].join(', ').  Please use #{self.class.name.sub('ActiveModel::Serializer', 'ActiveModelSerializers')}`  (not tested in terminal)